docs/articles_en/get-started/install-openvino/install-openvino-pip.rst

```diff
@@ -15,6 +15,19 @@ Install Intel® Distribution of OpenVINO™ Toolkit from PyPI Repository
    * is dedicated to users of all major OSes: Windows, Linux, and macOS
      (all x86_64 / arm64 architectures)
    * macOS offers support only for CPU inference
+   
+| **Simplified Build and Integration**  
+|   The package includes CMake configurations, precompiled static libraries, and headers, which
+    can be easily accessed through the Python API. You can use the `get_cmake_path()` method to
+    retrieve the paths to the CMake configurations and libraries:
+
+.. code-block:: python
+
+   from openvino import get_cmake_path
+   cmake_path = get_cmake_path()
+   
+For detailed instructions on how to use these configurations in your build setup, check out the
+:ref:`Create a library with extensions <create_a_library_with_extensions>` section.
```

Comment:
@tsavina Could you please check that this link is right? I'm not sure that correctly set it

Reply:
It looks fine :)
Also, we may later move this piece of text a bit lower, maybe under the "install" heading. But don't worry, we'll take care of it.

Reply:
yep, works just fine, you can also check it in the preview build https://openvino-doc.iotg.sclab.intel.com/ts-build-ov/get-started/install-openvino/install-openvino-pip.html